Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/datadonation/OneTimePasswordTest.kt

```diff
@@ -0,0 +1,14 @@
+package de.rki.coronawarnapp.datadonation
+
+import io.kotest.matchers.shouldBe
+import org.junit.Test
+import java.util.UUID
+
+class OneTimePasswordTest {
+
+    @Test
+    fun test() {
```

Comment:
Why is this an instrumentation test?

Reply:
because bytebuffer is only stub in android studio. obsolete with https://github.com/corona-warn-app/cwa-app-android/pull/2232#discussion_r566806424